src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Legacy/CSharpTokenizer.cs

```diff
@@ -561,8 +563,8 @@ private StateResult VerbatimStringLiteral()
 
     private StateResult QuotedStringLiteral() => QuotedLiteral('\"', IsEndQuotedStringLiteral, SyntaxKind.StringLiteral);
 
-    private readonly Func<char, bool> IsEndQuotedCharacterLiteral = (c) => c == '\\' || c == '\'' || ParserHelpers.IsNewLine(c);
-    private readonly Func<char, bool> IsEndQuotedStringLiteral = (c) => c == '\\' || c == '\"' || ParserHelpers.IsNewLine(c);
+    private readonly Func<char, bool> IsEndQuotedCharacterLiteral = (c) => c == '\\' || c == '\'' || SyntaxFacts.IsNewLine(c);
+    private readonly Func<char, bool> IsEndQuotedStringLiteral = (c) => c == '\\' || c == '\"' || SyntaxFacts.IsNewLine(c);
```

Comment:
Probably, and I'd be happy to do that as a follow up, but I want to keep this change only the MS.CA changes in case it needs to be reverted for whatever reason.

Reply:
https://github.com/dotnet/razor/pull/8934